libs/common-services/src/main/java/uk/gov/di/ipv/core/library/dto/CriConfig.java

```diff
@@ -1,14 +1,23 @@
 package uk.gov.di.ipv.core.library.dto;
 
 import com.nimbusds.jose.jwk.ECKey;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
 
 import java.net.URI;
 import java.text.ParseException;
 
-public interface CriConfig {
-    URI getCredentialUrl();
+@AllArgsConstructor
+@NoArgsConstructor
+@SuperBuilder
+public class CriConfig {
```

Comment:
Made this a class instead of an interface because implementing classes can just extend, while this class is the shape of the cri config for inherited identity